src/schoology-extractor/schoology_extractor/__main__.py

```diff
@@ -59,21 +82,7 @@ def _create_file_from_dataframe(action: Callable, file_name) -> bool:
         if data is not None:
             csv_writer.df_to_csv(data, file_name)
         return True
-    except Exception:
-        logger.exception("An exception occurred while generating %s", file_name)
-        return False
-
-
-# TODO: this method should disappear when we finish converting all of the output
-# to use the official CSV formats.
-def _create_file_from_list(action: Callable, file_name: str) -> bool:
-    try:
-        data = action()
-        if data is not None:
-            logger.info(f"Exporting {file_name}")
-            csv_writer.to_csv(data, os.path.join(schoology_output_path, file_name))
-        return True
-    except Exception:
+    except BaseException:
```

Comment:
`Exception` actually inherits from `BaseException`: "all exceptions must be instances of a class that derives from BaseException" ([source](https://docs.python.org/3/library/exceptions.html))